app/services/editions/EditionsTemplating.scala

```diff
@@ -156,11 +156,11 @@ class CollectionTemplatingHelper(capi: Capi, ophan: Ophan) extends Logging {
   private def getArticleItemsFromCapi(prefillParams: PrefillParamsAdapter): List[Prefill] = {
     // TODO: This being a try will hide a litany of failures, some of which we might want to surface
     try {
-      Await.result(capi.getUnsortedPrefillArticleItems(prefillParams), 10 seconds)
+     capi.getUnsortedPrefillArticleItems(prefillParams)
     } catch {
       case NonFatal(t) =>
         // At least log this as a warning so we can trace frequency
-        logger.warn(s"Failed to successfully execute CAPI prefill query $prefillParams", t)
+        logger.info(s"Failed to successfully execute CAPI prefill query $prefillParams", t)
```

Comment:
Why has this gone to info instead of warning?  Also see comment line above.

Reply:
sorry my bad will fix that